EMBER_Metadata/schema/README.md

```diff
@@ -0,0 +1,45 @@
+### Getting Started
```

Comment:
Could we add a quick section ### EMBER Project Metadata which describes the purpose of this repo in 2-3 sentences?